### ISSUE TITLE
A prop to control whether a parent item should expand itself on click

### ISSUE THREAD

**Issue by huayunh** [pxblue/react-component-library#234]
#### Describe the desired behavior

Add a prop `expandOnClick`: when click on a nested item parent with a landing page, does the panel expand or collapse itself. Default false.
Always true when a landing page is not present.

#### Describe the current behavior

On docit, when clicking on a parent nav item with a landing page, the panel does not expand by itself.

**Comment by huayunh** [comment]
A more detailed explanation:
![image](https://user-images.githubusercontent.com/8997218/106666118-2cac4b00-6575-11eb-94fb-e66ed4758699.png)

**Comment by jeffvg** [comment]
Investigate if the other frameworks will need this enhancement.

**Comment by joebochill** [comment]
Do we want to do some user testing around this before we jump into making a change like this?

**Comment by huayunh** [comment]
Good idea. Maybe not as serious as testing but certainly asking around devs / designers

**Comment by daileytj** [comment]
Adding this to the next milestone for now. @huayunh are we still wanting to do this?

**Comment by ektaghag-eaton** [comment]
Hi @huayunh , do we need this ?
Currently we have behaviour like below (you can check https://blui-react-patterns.web.app/action-list)

https://github.com/etn-ccis/blui-react-component-library/assets/133877691/5813505e-e75e-4f79-b561-77294d35a56b
